class BreakException(Exception):
    pass

class ContinueException(Exception):
    pass